fuzzer/xaac_enc_fuzzer.cpp

```diff
@@ -203,105 +203,16 @@ static VOID ixheaace_read_drc_config_params(
-        fuzzed_data->ConsumeBool();
-    if (1 == pstr_enc_loudness_info_set->str_loudness_info[n].true_peak_level_present) {
-      pstr_enc_loudness_info_set->str_loudness_info[n].true_peak_level =
-          fuzzed_data->ConsumeFloatingPoint<FLOAT32>();
-      pstr_enc_loudness_info_set->str_loudness_info[n].true_peak_level_measurement_system =
-          fuzzed_data->ConsumeIntegral<WORD8>();
-      pstr_enc_loudness_info_set->str_loudness_info[n].true_peak_level_reliability =
-          fuzzed_data->ConsumeIntegral<WORD8>();
-    }
-
-    pstr_enc_loudness_info_set->str_loudness_info[n].measurement_count =
-        fuzzed_data->ConsumeIntegralInRange<WORD8>(0, MAX_MEASUREMENT_COUNT);
-    for (m = 0; m < pstr_enc_loudness_info_set->str_loudness_info[n].measurement_count; m++) {
-      pstr_enc_loudness_info_set->str_loudness_info[n].str_loudness_measure[m].method_definition =
-          fuzzed_data->ConsumeIntegral<WORD8>();
-      pstr_enc_loudness_info_set->str_loudness_info[n].str_loudness_measure[m].method_value =
-          fuzzed_data->ConsumeFloatingPoint<FLOAT32>();
-      pstr_enc_loudness_info_set->str_loudness_info[n]
-          .str_loudness_measure[m]
-          .measurement_system = fuzzed_data->ConsumeIntegral<WORD8>();
-      pstr_enc_loudness_info_set->str_loudness_info[n].str_loudness_measure[m].reliability =
-          fuzzed_data->ConsumeIntegral<WORD8>();
-    }
-  }
-
-  pstr_enc_loudness_info_set->loudness_info_album_count =
-      fuzzed_data->ConsumeIntegralInRange<WORD8>(0, MAX_LOUDNESS_INFO_COUNT);
-  for (n = 0; n < pstr_enc_loudness_info_set->loudness_info_album_count; n++) {
-    pstr_enc_loudness_info_set->str_loudness_info_album[n].drc_set_id =
-        fuzzed_data->ConsumeIntegral<WORD8>();
-    pstr_enc_loudness_info_set->str_loudness_info_album[n].downmix_id =
-        fuzzed_data->ConsumeIntegral<WORD8>();
-    pstr_enc_loudness_info_set->str_loudness_info_album[n].sample_peak_level_present =
-        fuzzed_data->ConsumeBool();
-    if (1 == pstr_enc_loudness_info_set->str_loudness_info_album[n].sample_peak_level_present) {
-      pstr_enc_loudness_info_set->str_loudness_info_album[n].sample_peak_level =
-          fuzzed_data->ConsumeFloatingPoint<FLOAT32>();
-    }
-    pstr_enc_loudness_info_set->str_loudness_info_album[n].true_peak_level_present =
-        fuzzed_data->ConsumeBool();
-    if (1 == pstr_enc_loudness_info_set->str_loudness_info_album[n].true_peak_level_present) {
-      pstr_enc_loudness_info_set->str_loudness_info_album[n].true_peak_level =
-          fuzzed_data->ConsumeFloatingPoint<FLOAT32>();
-      pstr_enc_loudness_info_set->str_loudness_info_album[n].true_peak_level_measurement_system =
-          fuzzed_data->ConsumeIntegral<WORD8>();
-      pstr_enc_loudness_info_set->str_loudness_info_album[n].true_peak_level_reliability =
-          fuzzed_data->ConsumeIntegral<WORD8>();
-    }
+  pstr_uni_drc_config->str_channel_layout.layout_signaling_present = 0;
```

Comment:
this change seems wrong.

with this drc path will not be tested well.

please revert this change.

Reply:
@akshayragir833 @divya-bm @sandeshvenkatesh